kong/conf_loader.lua

```diff
@@ -61,6 +61,8 @@ local CONF_INFERENCES = {
   cluster_advertise = {typ = "string"},
   nginx_worker_processes = {typ = "string"},
   upstream_keepalive = {typ = "number"},
+  server_tokens = {typ = "boolean"},
+  latency_tokens = {typ = "boolean"},
 
```

Comment:
`server_header` is kinda bad as it says "no headers at all". Something like `kong_version_headers` and `kong_latency_headers` could be better.